### PR TITLE
fix mapbox-v4

### PR DESCRIPTION
**Pull request by kespindler**
Resolves issues from #860 and #862 .

### CODE REVIEW

File: flask_admin/static/admin/js/form-1.0.0.js [hunk 1]
```diff
@@ -78,10 +78,6 @@
           return false;
         }
 
-        if (!window.MAPBOX_ACCESS_TOKEN) {
```
Comment (mrjoes) [inline flask_admin/static/admin/js/form-1.0.0.js]: If there's a way to check version on client-side and spit out warning if access token is missing?

**Comment by kespindler** [comment]
It's a good point - that's the philosophical problem with removing the MAPBOX_API_VERSION field. For full clarity, I think re-introducing that is a good thing, and I can re-add to PR.

**Comment by mrjoes** [comment]
No, don't add server-side MAPBOX_API_VERSION configuration setting. I bet there's way to check on client-side what's mapbox version and only then check if token is provided.

**Comment by nitishr** [comment]
I apologize for the copy and paste errors in my earlier patch. Will be a lot more careful next time.

I've verified this fix locally and it works fine. Thanks @kespindler. Can this be merged, since without this, the map integration remains broken for mapbox API v4 users.

**Comment by kespindler** [comment]
@nitishr Yup, it works fine on my end too. Merge at will!

**Pull request merged**